requirements3.txt

```diff
@@ -9,4 +9,4 @@ PyYAML==5.3
 ruamel.yaml==0.16.5
 setuptools==44.0.0
 sh==1.12.14
-singledispatch==3.4.0.3
+singledispatch==3.4.0.3
```

Comment:
I don't quite understand why this is considered a change, I'm afraid that there is some encoding thing going on, can you revert changes in this file?